client/utils/log-error.js

```diff
@@ -1,10 +1,14 @@
 const logError = (error, errorText = "") => {
-  console.error(
-    'Status',
-    error.response.status,
-    error.response.statusText,
-    ':',
-    errorText
-  );
+  try {
```

Comment:
added this try-catch block because in the latest version of node (v15.0.1),  there were instances where `error` didn't have the attribute `response` causing it to raise the error complained by `Travis buddy` above.